README.md

```diff
@@ -15,6 +15,83 @@ exclude bundled Android dependencies.
 android.jetifier.ignorelist=android-base-common,common
 ```
 
+Git LFS
+--------
+It is recommended you use [Git LFS][lfs] to store your snapshots.  Here's a template to get started.
+
+#### **`.gitattributes`**
+```bash
+**/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
+```
+
+#### **`.hooks/pre-receive`**
+```bash
+#!/bin/bash
```

Comment:
nit: either omit this as implied or use `/usr/bin/env bash` which is the correct way to look up bash.